engine/lib/cron.php

```diff
@@ -75,15 +72,22 @@ function _elgg_cron_run() {
 		$key = "cron_latest:$period:ts";
 		$ts = elgg_get_site_entity()->getPrivateSetting($key);
 		$deadline = $ts + $interval;
+
 		if ($now > $deadline) {
-			$old_stdout = elgg_trigger_plugin_hook('cron', $period, $params, $old_stdout);
```

Comment:
Not sure how to deal with $return value here. If a plugin checks for return value before doing something, they are getting the wrong message that was added by another period... So, it's a bugfix, I suppose.